modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) works on your cluster
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `lksctp-tools` package.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a Pod that runs an SCTP listener:
+
+.. Create a file named `sctp-server.yaml` that defines a Pod using the following YAML:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: sctpdemo
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  containers:
+    - name: sctpserver
+      image: fedora:31
+      command: ["/bin/sh", "-c"]
+      args:
+       ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]
+      ports:
+        - containerPort: 30100
+          name: sctpserver
+          protocol: SCTP
+----
+
+.. Create the Pod by running the following command:
++
+----
+$ create -f sctp-server.yaml
+----
+
+. Create a Service for the SCTP listener Pod.
+
+.. Create a file named `sctp-service.yaml` that defines a Service using the following YAML:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: sctpdemo
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  selector:
+    app: sctpserver
+  ports:
+    - name: sctpserver
+      protocol: SCTP
+      port: 30100
+      targetPort: 30100
+  type: ClusterIP
+----
+
+.. Create the Service by running the following command:
++
+----
+$ create -f sctp-service.yaml
+----
+
+. Run the following command to connect to the SCTP listener:
++
+----
+$ sctp_test -H localhost -P 30100 -l
```

Comment:
This command is to run the sctp server, we need example that shows how to launch deployment with one pod from where user can fire sctp client to connect to this server. @danwinship @trozet do you have instructions to start pod and run sctp sample application, otherwise i will dig into the ci tests to find something.

Reply:
I found a good example which can fire both sctp server pod and sctp client in https://bugzilla.redhat.com/show_bug.cgi?id=1796157, the configuration steps are from @fedepaol who can comment on this

Reply:
The instructions on the bz are fine. They were for the default namespace but they can be modified to apply to a custom namespace.

Reply:
@vishnoianil @fedepaol 

Should a customer install an image from quay.io? Or is there a different image that makes sense?

Should this be done in a different namespace than `default`?

Thanks!

Reply:
I asked @sferich888, and we can't use the `quay.io` image for this -- Is there a different way to validate that uses a Red Hat supported image?

Reply:
Sorry for the late reply. 
Just plain fedora is fine I think. It is what I used in the kba.

Reply:
Or any other image that has installed lskctp-tools.

Reply:
@fedepaol, do you know how to find images that might include `lskctp-tools`? Fedora isn't permissible, either.

Could I just log in to a node and confirm `sctp` is loaded in by running `lsmod`?

Reply:
@jboxman this needs to be changed I think. Once you have the server pod running, you need to spin up a new pod (client) to which you bash into and try to connect to the server (as described in https://bugzilla.redhat.com/show_bug.cgi?id=1796157 ) 
The pod description is the one named `sctpclient`. Client connection instructions in:
`5. Connect to the client pod and launch sctp_test in client mode, using the server pod's address:`

Reply:
@fedepaol, okay, I didn't realize a second Pod was still necessary for this.